nin/backend/generate/templateShader/uniforms.json

```diff
@@ -0,0 +1,3 @@
+{
+    "time": { "type": "f", "value": null }
```

Comment:
I think we should have tDiffuse in uniforms for the default shader.

I ran into some very weird issues when I forgot to put it in there.